server/models/Request.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+   		ref: 'profile'
+    },
+    setter: {
```

Comment:
what's the setter in this scenario? would it be the pet sitter requesting or would it be someone who is setting the request?

Reply:
It's suppose to be sitter typing error